RocketControlUnitGUI/src/lib/stores.ts

```diff
@@ -3,6 +3,14 @@ import { writable, type Writable } from 'svelte/store';
 export const currentState = writable('N/A');
 export const auth = writable(false);
 
+// These are intended for the smaller mini RCU screen.
+// When `miniRcu` is enabled, the sidebar can be
+// collapsed and expanded by clicking the soar logo.
+// Additionally, there are confirmation modals
+// when clicking the toggle switches.
+export const miniRcu = writable(true);
```

Comment:
how are we enabling this? I think it might be best to detect this based on screen size.

Reply:
Yea I can make this switch on at a certain screen size once we finalize the decision for which screen we go with.